### PR TITLE
Updated long message for fn command

### PR DESCRIPTION
**Pull request by neha-viswanathan**
Updated long message for `function` [alias: `fn`] command

### CODE REVIEW

File: cmd/amp/function.go [hunk 1]
```diff
@@ -26,6 +27,8 @@ var (
 	createFunctionCmd = &cobra.Command{
 		Use:   "create NAME IMAGE",
 		Short: "Create a function",
+		Long: `Create command registers a function into ETCD, with the specified name and image.
```
Comment (subfuzion) [inline cmd/amp/function.go]: The create command registers a function with the specified name and image. If successful, a function ID is returned.

File: cmd/amp/function.go [hunk 2]
```diff
@@ -34,6 +37,7 @@ var (
 	listFunctionCmd = &cobra.Command{
 		Use:   "ls",
 		Short: "List functions",
+		Long:  `List command displays all the functions available in ETCD.`,
```
Comment (subfuzion) [inline cmd/amp/function.go]: The list command displays all registered functions.

File: cmd/amp/function.go [hunk 3]
```diff
@@ -42,6 +46,7 @@ var (
 	removeFunctionCmd = &cobra.Command{
 		Use:   "rm FUNCTION",
 		Short: "Remove a function",
+		Long:  `Remove command unregisters the specified function id from ETCD.`,
```
Comment (subfuzion) [inline cmd/amp/function.go]: The remove command unregisters the specified function.

File: cmd/amp/function.go [hunk 4]
```diff
@@ -42,6 +46,7 @@ var (
 	removeFunctionCmd = &cobra.Command{
 		Use:   "rm FUNCTION",
```
Comment (subfuzion) [inline cmd/amp/function.go]: FUNCTION should really be something `FUNC-NAME-OR-ID` but I don't think we support name right now (or even ID fragments, like the other commands. To make it clear, we should probably say FUNC-ID for now.

**Review by subfuzion — CHANGES_REQUESTED** [review]

File: cmd/amp/function.go (hunk 3 above)
Comment (neha-viswanathan) [reply cmd/amp/function.go]: Done.

File: cmd/amp/function.go (hunk 4 above)
Comment (neha-viswanathan) [reply cmd/amp/function.go]: Done.

File: cmd/amp/function.go (hunk 2 above)
Comment (neha-viswanathan) [reply cmd/amp/function.go]: Done.

File: cmd/amp/function.go (hunk 1 above)
Comment (neha-viswanathan) [reply cmd/amp/function.go]: Done.

**Review by subfuzion — APPROVED** [review]
👍

**Pull request merged**